### PR TITLE
Remove call to retrieve token 

### PR DESCRIPTION
**Pull request by awhodgson256**
## What does this pull request do?

Remove call to retrieve token - addressing issue with nDelius integration

## What is the intent behind these changes?

Fix integration with nDelius on referral send

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RamDeliusAPIConvictionService.kt [hunk 1]
```diff
@@ -41,13 +39,7 @@ class RamDeliusAPIConvictionService(
...
       .toString()
 
-    val auth = SecurityContextHolder.getContext()?.getAuthentication()
-    var token: JwtAuthenticationToken? = null
-    if (auth !== null) {
-      token = auth as JwtAuthenticationToken
-    }
-
-    val convictionDetails = ramDeliusApiClient.makeGetConvictionRequest(convictionPath, token)
+    val convictionDetails = ramDeliusApiClient.makeGetConvictionRequest(convictionPath, null)
```
Comment (kth13) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RamDeliusAPIConvictionService.kt:42]: @awhodgson256 could you please default the null value in `ramDeliusApiClient.makeGetConvictionRequest(convictionPath, null)`

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/RamDeliusClient.kt [hunk 2]
```diff
@@ -14,6 +13,6 @@ class RamDeliusClient(private val ramDeliusApiClient: RestClient) {
   fun makePutAppointmentRequest(uri: String, requestBody: Any): AppointmentResponseDTO? =
     ramDeliusApiClient.put(uri, requestBody).retrieve().bodyToMono(AppointmentResponseDTO::class.java).block()
 
-  fun makeGetConvictionRequest(uri: String, token: JwtAuthenticationToken?): ConvictionDetails =
-    ramDeliusApiClient.get(uri, null, token).retrieve().bodyToMono(ConvictionDetails::class.java).block()
+  fun makeGetConvictionRequest(uri: String): ConvictionDetails =
+    ramDeliusApiClient.get(uri, null, null).retrieve().bodyToMono(ConvictionDetails::class.java).block()
```
Comment (kth13) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/RamDeliusClient.kt:17]: @awhodgson256 The last two parameters are already defaulted to null. So we don't need to pass that as well `ramDeliusApiClient.get(uri, null, null)`

**Review by kth13 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RamDeliusAPIConvictionService.kt (hunk 1 above)
Comment (awhodgson256) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RamDeliusAPIConvictionService.kt]: Rectified that @kth13

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/RamDeliusClient.kt (hunk 2 above)
Comment (awhodgson256) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/RamDeliusClient.kt]: @kth13 removed to simplify as suggested

**Review by kth13 — APPROVED** [review]
Approved

**Pull request merged**